index.rst

```diff
@@ -354,3 +356,4 @@ Full Table of Contents
 .. _IronPython: http://ironpython.net/
 .. _Stackless: http://www.stackless.com/
 .. _Issue tracker: https://bugs.python.org/
+.. _Open Reviews: https://github.com/python/cpython/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3A%22awaiting%20review%22
```

Comment:
This should now be changed, from `.. _Open Reviews: ` to `.. _open pull requests:`.